tests/README.md

```diff
@@ -1,10 +1,9 @@
 # Testing
 There are two types of tests used:
-1. **functional tests** (`functional/`) - this folder contains C code for functional tests used internally in Topic Square,
+1. **functional tests** (`functional/`) - this folder contains C code for functional tests used internally in Tropic Square, which test libtropic core API,
```

Comment:
maybe something like this?

... - this folder contains C code of functional tests which verify libtropic core API.

idk whether we need to emphasize that they are used internally. They can be used by anyone with an access to model.